src/classes/ADDR_Addresses_TEST.cls

```diff
@@ -1579,4 +1694,99 @@ public with sharing class ADDR_Addresses_TEST {
             system.assertEquals(!fSeasonal, addr.Default_Address__c);
         }        
     }   
+
+    // Helpers
+    ////////////
+
+    /*********************************************************************************************************
+    * @description Populates Contact's Mailing Address from the Address
+    * @param c Contact
+    * @param sourceAddress Address
+    * @return Contact
+    **********************************************************************************************************/
+    static Contact populateContactAddress(Contact c, Address__c sourceAddress) {
```

Comment:
What's the reason for returning the contact? Can't you return void and the contact in the parameter will be updated anyway?